### PR TITLE
Marcus review

### PR DESCRIPTION
**Pull request by marcusphillips**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -377,6 +449,14 @@ <h3>Defining Zome Functions</h3>
...
         </section>
 
+        <section>
+          <fragment language="markdown" animate="by-line with-ancestry">
+            ### Entry Retrievals
+            #### Overview
+
+            - Read requests issued to the DHT
+          </fragment>
+
```
Comment (guillemcordoba) [inline index.html:459]: We are missing a closing "</section>" here.

File: index.html [hunk 2]
```diff
@@ -252,9 +284,10 @@ <h3>Defining Zome Functions</h3>
 
         <section>
           <fragment language="markdown" animate="by-line with-ancestry">
-            ### Defining Entries
+            ### Defining An Entry Format
```
Comment (guillemcordoba) [inline index.html:287]: Just FYI, the term that the scaffolding tool and the developers docs have for this is "Entry Type".

File: index.html [hunk 3]
```diff
@@ -354,11 +427,10 @@ <h3>Defining Zome Functions</h3>
             #### delete_entry()
 
             - Must be scoped to an action, not an entry
-                - It may be ambiguous which of several valid actions pointing to the same entry you intend to delete
             - Deleted entry and action will still exist in the DHT after deleting
-                - We are just marking them as "deleted"
+              - We are just marking them as "deleted"
             - One action may be deleted multiple times
-            - It's up to the application to interpret what "deleting an action" means
+            - It's up to the application to interpret what it "deleting an action" means
```
[the remark this answers is not in the document]
Comment (marcusphillips) [reply index.html:435]: correct!

File: index.html (hunk 1 above)
Comment (marcusphillips) [reply index.html:461]: I'm not seeing it... Where was it opened? @mmeigooni if you see it could you fix it?

File: index.html (hunk 2 above)
Comment (marcusphillips) [reply index.html:289]: Fantastic-- @mmeigooni my local repo is in some funky state I can't figure out quickly. I'll try to do a deep debug soon, but until then would you please resolve any issues that we complete discussions on and merge?

File: index.html (hunk 1 above)
Comment (marcusphillips) [reply index.html:461]: I'm not seeing it... Where was it opened? @mmeigooni if you see it could you fix it?

Comment (marcusphillips) [reply index.html:461]: @guillemcordoba I was not able to locate the issue you're referring to here. Have to merge this for tomorrow. Please push an edit when you get a moment?

File: index.html (hunk 2 above)
Comment (marcusphillips) [reply index.html:289]: handled

**Pull request merged**